### ISSUE TITLE
Delete list items

### ISSUE THREAD

**Issue by SaschaRadojkovic** [SaschaRadojkovic/capstone-project#3]
## Value Proposition

**As a** User I want to have a delete button on every list item

**so that** i can delete items from the list
​

## Description

![image](https://user-images.githubusercontent.com/16897094/213684542-08971e62-933f-4e7b-9d37-652657d28eed.png)

## Acceptance Criteria

- [ ] Each item in the list has a delete button
- [ ] The user can delete an item by clicking the delete button
- [ ] The list updates in real-time to reflect the deletion

      ​

## Tasks

- [ ] Add a delete button next to each item in the list
- [ ] Implement functionality to handle the click event of the delete button

      ​

## Complexity

... (T-Shirt Sizes S)

**Comment by NikRoe** [comment]
Hi Sascha,

I have a few suggestions / notes:

I would rephrase the value proposition to something more precise, like:

As a User
I want to have a search bar
so that i can find allergenes to avoid

In your acceptance criteria you should be more specific about what will happen when interacting with the search bar:
- what happens if no entry is found?
- will there be a mininum / maximum search entry length?
- will there be suggestions while searching or will I need to submit the search form to see results?

Will you have a component for your search bar? If so, mention it in your user story

From your wireframe I am not certain where the data will be displayed after searching for a term. 
Will only the search results appear or will they be displayed below the rest of the page? Please specify :) 

I am also not sure about the purpose of the "allergenes" button. If that is a button? What does this element do for the moment?

With reference to your first user story: The checkboxes have no functionality yet, so you can leave them out for now

The scope of your user story is very nice, though, and is exactly what we are aiming for :)

**Comment by RobinWitt** [comment]
Good evening,
is the 'allergens' button there to be able to switch between 'settings' and your 'allergens' page, or is this page part of the 'settings' group?

Did you make sure that there are no allergens shorter than four or longer than 20 digits? If you make an option to show 'no results' and also display suggestions, you may not need the length restriction on your search. 
('egg' came to my mind, for example)

Your second-to-last acceptance criteria is not really clear to me, I wouldn't know what to do with that as a developer. Maybe you can make it more descriptive?

I hope this feedback is helpful to you.
